tests/components/charts.test.ts

```diff
@@ -186,6 +186,18 @@ describe("figures", () => {
     }
   );
 
+  test.each(["scorecard", "basicChart", "gauge"])(
+    "Double click on graph will open sidepanel",
+    async (chartType: string) => {
+      createTestChart(chartType);
+      await nextTick();
+      expect(document.querySelector(".o-chart-container")).toBeTruthy();
+      triggerMouseEvent(".o-chart-container", "dblclick");
+      await nextTick();
+      expect(document.querySelector(".o-sidePanel")).toBeTruthy();
```

Comment:
Also check that the chart is selected. Looks good otherwise 👍

Reply:
Done